### PR TITLE
Gh enterprise 1279

### PR DESCRIPTION
**Pull request by danantonioreyes**

### CODE REVIEW

**Comment by bendk** [comment]
We talked about this one on the dev call a bit.  I think it would be cleaner if:
  - Each Action object gets a requires_subtitle_language_tip attribute.  By default, it can be true.  For SaveDraft it will be false.
  - action_requires_subtitle_language_tip just checks that value.

The reason I like that it it seems easier to change the behavior more in the future.  With the current system, we will always need to update all SubtitleWorkflow objects.

**Comment by bendk** [comment]
I think it looks good, but can you try merging master in amara-enterprise?  There's a conflict there, and also I see a couple extra commits.

File: apps/subtitles/workflows.py [hunk 1]
```diff
@@ -835,6 +840,7 @@ class SaveDraft(Action):
     in_progress_text = ugettext_lazy('Saving')
     complete = None
     requires_translated_metadata_if_enabled = False
+    requires_subtitle_language_tip = False
```
Comment (bendk) [inline apps/subtitles/workflows.py]: I think save draft needs a subtitle language tip, otherwise what are you saving?

**Comment by danantonioreyes** [comment]
Should be good now, updated with dev and submodules

**Comment by bendk** [comment]
Looks great, sorry for all the merge issues.

**Pull request merged**